src/components/VaccinationsTable/VaccinationsTable.stories.tsx

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
+import { MapView } from 'screens/HomePage/HomePage';
```

Comment:
FYI- My map PR moves this enum to a different file.  Guessing you'll merge first but if not, this will need to change to:

```
import { MapView } from 'screens/HomePage/utils';
```